v2/contribute/checklists/saas/db-restore.md

```diff
@@ -0,0 +1,48 @@
+## Restoring API/Saas related DBs using automated backups
+
+### Database - Development API Server
+
+- get the latest automated backup of the production server db
+- use restore snapshot option
+- make sure while restoring, the db instance type, security groups and all other configuration (including the root password) are same as the old db cluster
+- update the route 53 entry related to this db (dev-mysql.supertokens.dns)
+- once the restoration is finished, update the following:
+    - development node api server config
+- restart development node api server containers
+- remove old cluster
+
+### Database - Production API Server
+
+- get the latest automated backup of the production server db
+- use restore snapshot option
+- make sure while restoring, the db instance type, security groups and all other configuration (including the root password) are same as the old db cluster
+- update the route 53 entry related to this db (production-mysql.supertokens.dns)
+- once the restoration is finished, update the following:
+    - production node api server config
+    - db configuration on retool site
+    - db configuration on metabase site
+- restart production node api server containers
+- remove old cluster
+
+### Database - API Saas DB
+
+- get the latest automated backup of the saas db
+- use restore snapshot option
+- make sure while restoring, the db instance type, security groups and all other configuration (including the root password) are same as the old db cluster
+- update the route 53 entry related to this db (production-saas-users-mysql.supertokens.dns)
+- once the restoration is finished, update the following:
+    - production node api server config
+- restart production node api server containers
+- remove old cluster
+
```

Comment:
No change to the sidebar? How did you even test this?!

Reply:
What do you mean by sidebar?